projects/addon-table/components/table/directives/test/table-sorting-fix.directive.spec.ts

```diff
@@ -0,0 +1,124 @@
+import {ChangeDetectionStrategy, Component, ViewChild} from '@angular/core';
```

Comment:
Let's drop this file completely, I don't think it's helpful and we will drop individual outputs in v5 anyway